scripts/ci/_utils.sh

```diff
@@ -885,3 +901,32 @@ function rebuild_all_images_if_needed_and_confirmed() {
         fi
     fi
 }
+
+function build_image_on_ci() {
+    if [[ "${CI:=}" != "true" ]]; then
+        print_info
+        print_info "Cleaning up docker installation!!!!!!"
+        print_info
+        "${AIRFLOW_SOURCES}/confirm" "Cleaning docker data and rebuilding"
+    fi
+
+    export AIRFLOW_CONTAINER_FORCE_PULL_IMAGES="true"
+    export FORCE_BUILD="true"
+    export VERBOSE="${VERBOSE:="false"}"
+
+    # Cleanup docker installation. It should be empty in CI but let's not risk
+    docker system prune --all --force
+    rm -rf "${BUILD_CACHE_DIR}"
+
+    if [[ ${TRAVIS_JOB_NAME:=""} == "Tests"* ]]; then
+        rebuild_ci_image_if_needed
+    elif [[ ${TRAVIS_JOB_NAME} == "Check lic"* ]]; then
+        rebuild_checklicence_image_if_needed
+    else
```

Comment:
Can you specify the cases when this image is build and the other case returns an error? This may allow easier script debugging.

Reply:
Fixed :)